Rules/UseConsistentIndentation.cs

```diff
@@ -99,7 +99,21 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                         // we add this redundant check
                         if (onNewLine)
                         {
-                            AddViolation(token, indentationLevel, diagnosticRecords, ref onNewLine);
+                            var tempIndentationLevel = indentationLevel;
+
+                            // Ignore comments
+                            int j = k - 2;
```

Comment:
What's the `- 2` indicate?  For "magic numbers" like this, it'd be good to leave a comment to say why it works